packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -55,17 +56,15 @@ export interface InstalledPackageInfo {
 
 export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
   public readonly relativeMetdataTempPath = path.join(
-    '.sfdx',
-    'tools',
+    projectPaths.relativeToolsFolder(),
     'isvdebuggermdapitmp'
   );
   public readonly relativeApexPackageXmlPath = path.join(
     this.relativeMetdataTempPath,
     'package.xml'
   );
   public readonly relativeInstalledPackagesPath = path.join(
-    '.sfdx',
-    'tools',
+    projectPaths.relativeToolsFolder(),
```

Comment:
I realize adding a full blown jest unit test suite for this file is probably more lift then is worthwhile for the work being addressed, but you would add a test(s) to verify the `relativeInstalledPackagesPath` and `relativeMetdataTempPath` readonly values are set appropriately.